tools/test-case-generators/format-request-map.json

```diff
@@ -717,6 +734,92 @@
       }
     },
     "overrides": {
+      "rhel-91": {
```

Comment:
They will, I was just too lazy to copy-paste the overrides for all since they use the same pipelines, it should be fine.  When RHEL 8 gets the feature, we can remove the override and make it part of the standard `with-customizations` variant.

Reply:
I see. It would be probably much easier if we could use wildcards in the distro name in `format-request-map.json`...